modules/git-workflow-github.md

```diff
@@ -270,14 +273,19 @@ at (doing a code review), and approve a good solution.
   - Describe the targets in the PR body so that readers can understand
     in more details what the PR is about
   - If your PR resolves an issue just add `Resolves #3` somewhere 
-    in the comment (`3`
-    is a number of an issue you can see beside its title). GitHub will
-    bind your PR to that issue and the issue will get closed
+    in the comment (`3` is a number of an issue you can see beside its title). 
+    GitHub will bind your PR to that issue and the issue will get closed
     (as resolved) once your PR is merged.
 - Draw attention
   - PR created from a fork is bound to the original repo by GitHub,
     so navigate to the original repo, Pull Requests and open a PR
-    you've opened
+    you've opened  
+```
+    You may not assign reviewers if you're not an owner of the repo
+    or not a part of organization. Try identifying the owner or repo
+    collaborators and mention them in a comment by their names.
+    If unable to identify relevant persons then just skip this step.
+```
```

Comment:
Is there any good reason to make styling of this block so much different from surrounding?
You can pre-view the render using built-in GitHub tools:
![image](https://user-images.githubusercontent.com/8686631/66628610-fd4b2f80-ec07-11e9-87fa-5f2e8d3cc1ef.png)

Suggested change renders like this:
![image](https://user-images.githubusercontent.com/8686631/66628666-1d7aee80-ec08-11e9-8dc6-ca0fa211bdcd.png)

Would you please keep the styling consistent?

Reply:
It might be though a good idea to draw special attention at this point. What about markdown blockquoting? Did you try it?